packages/pdf-renderer/src/lib/pdfGenerator.service.ts

```diff
@@ -70,12 +70,7 @@ export class PdfGenerator {
         await PdfGenerator.preparePageForScreenshot(page, data);
 
         return (await page.screenshot({
-            captureBeyondViewport: data.captureBeyondViewport,
-            type: data.type,
-            fullPage: data.fullPage,
-            quality: data.quality,
-            clip: data.clip,
-            fromSurface: data.fromSurface,
+            ...data,
         })) as Buffer;
```

Comment:
Why not just pass `data` as is ? :thinking: Are they being mutate by `page.screenshot`?